stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-17.5
+resolver: lts-16.31
```

Comment:
I assume you're only changing this so that CI will run on lts-16, and that you don't intend to merge this PR, right?

Reply:
That's right!